Utilities/bootstrap

```diff
@@ -710,13 +737,19 @@ def get_swift_build_tool_path():
                 return sbt_path
         except subprocess.CalledProcessError:
             pass
-    else:
+    elif platform.system() == 'Linux':
         # Otherwise, search for it in PATH.
         try:
             return subprocess.check_output(["which", "swift-build-tool"],
                                            universal_newlines=True).strip()
         except:
             pass
+    elif platform.system() == 'Windows':
```

Comment:
Put the Windows case above the Linux case, this style breaks (for example) FreeBSD unnecessarily.